R/model_parameters.merMod.R

```diff
@@ -21,12 +21,13 @@ model_parameters.merMod <- function(model, ci = .95, standardize = FALSE, bootst
   parameters <- .extract_parameters_mixed(model, ci = ci, p_method = p_method, ci_method = ci_method, ...)
 
   # Standardized
-  if (standardize) {
-    std_model <- standardize(model, ...)
-    std_parameters <- .extract_parameters_mixed(std_model, ci = ci, p_method = p_method, ci_method = ci_method, ...)
-    names(std_parameters) <- paste0("Std_", names(std_parameters))
-
-    parameters <- cbind(parameters, std_parameters[c("Std_beta", "Std_SE")])
+  # Standardized
+  if (standardize != FALSE & !is.null(standardize)) {
+    if(standardize == TRUE){
+      warning("Please set the `standardize` method explicitly. Set to \"refit\" by default.")
+      standardize <- "refit"
+    }
+    parameters <- cbind(parameters, standardize_parameters(model, method = standardize)[2])
```

Comment:
I mainly revised these code-chunks into:

```r
  # Standardized
  if (isTRUE(standardize)) {
    warning("Please set the `standardize` method explicitly. Set to \"refit\" by default.")
    standardize <- "refit"
  }

  if (!is.null(standardize) && !is.logical(standardize)) {
    parameters <- cbind(parameters, standardize_parameters(model, method = standardize)[2])
  }
```

`isTRUE()` captures also the "logical > 1 coercion"-problem in if-statements. And if `standardize = FALSE`, the 2nd if-condition is not true, so it should work as expected.

Reply:
great